inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -87,9 +89,14 @@ def fuzzy_match(obj, eng):
     """
     if not current_app.config.get('FEATURE_FLAG_ENABLE_FUZZY_MATCHER'):
         return False
-
+    math_ml_latex_regex = r"(<math(.*?)<\/math>|(?<!\\)\$.*?(?<!\\)\$|(?<!\\)\\(.*?(?<!\\)\\)|(?<!\\)\\[.*?(?<!\\)\\])"
     fuzzy_match_config = current_app.config['FUZZY_MATCH']
-    matches = dedupe_list(match(obj.data, fuzzy_match_config))
+    obj_data_without_math_ml_latex = copy(obj.data)
+    if "abstract" in obj.data:
+        abstract_dup = deepcopy(obj.data["abstracts"])
```

Comment:
you're never assigning this back to the `obj.data["abstracts"]`. How come the test passes then?